packages/wrangler/src/index.tsx

```diff
@@ -806,9 +815,10 @@ export async function main(argv: string[]): Promise<void> {
 
       const { waitUntilExit } = render(
         <Dev
-          name={args.name || config.name}
+          name={args.name ?? config.name}
```

Comment:
This actually subtly changes the behaviour here, right?
Previously `wrangler dev src/index.ts --name` would result in the name being found in the config, whereas now the name will be the empty string.
I think that we probably should error if the `--name` is used without a value, right?

Reply:
Yeah arguably this is better behaviour, but we still need actual validation of the values.